Makefile.am

```diff
@@ -41,10 +41,10 @@
 # OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #
 
-WARN_CFLAGS = -Wall -Wextra -Werror
+WARN_CFLAGS = -Wall -Wextra -Wformat=2 -Werror
 
 ACLOCAL_AMFLAGS = -I cmulocal
-AM_CFLAGS = $(WARN_CFLAGS) @PERL_CCCDLFLAGS@ $(GCOV_CFLAGS)
+AM_CFLAGS = -O3 $(WARN_CFLAGS) @PERL_CCCDLFLAGS@ $(GCOV_CFLAGS)
```

Comment:
Does this force us to always compile with `-O3` which we didn't before? It reduces the debugability of our builds quite considerably to do that.

Reply:
This is a temporary commit not to be included with the final bit